common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -45,6 +45,7 @@ import (
 	p "github.com/uber/cadence/common/persistence"
 	"github.com/uber/cadence/common/persistence/cassandra"
 	"github.com/uber/cadence/common/persistence/client"
+	_ "github.com/uber/cadence/common/persistence/nosql/nosqlplugin/cassandra/gocql/native" // needed to load the default gocql client
```

Comment:
so it means we can't reuse the tests for testing our internal gocql client impl?

maybe we can put it in `cassandra_test.go` so that gocql client only get registered in the open source test?